src/main/java/org/isf/opd/manager/OpdBrowserManager.java

```diff
@@ -66,7 +73,7 @@ protected void setPatientConsistency(Opd opd) {
 	 * @param insert {@code true} or updated {@code false}
 	 * @throws OHDataValidationException
 	 */
-	protected void validateOpd(Opd opd, boolean insert) throws OHDataValidationException {
+	public void validateOpd(Opd opd, boolean insert) throws OHDataValidationException {
```

Comment:
so far, "validate" methods are defined `protected` and never tested, should we start from this PR?

Reply:
The 'validate' methods are tested; they just require a lot more setup and infrastructure.   I was trying to show explicitly the testing of the new validation requirements.

BTW, the new unit testing setup that I proposed would "fix" this issue by not add `.test` to the package names.  That would allow tests to access `protected` methods directly.

In the short run the simple PR would be to remove `.test` from all the test package names.

Reply:
Yes, what I meant is that validate methods are not tested directly so far.

I'm working on the Admission validation tests as well (OP-1136), and I think we will end up with a unique approach, but in a different issue as you was suggesting in OP-911, leveraging for sure the `protected` modifier.